In the target case, we can't check if perl exists or try running it. If we
don't patch the code, the native perl location would be used, not the target
which is also incorrect. We therefore disable this code and rely on the 
correct value of PERL being passed in.

RP 31/8/2011

Upstream-Status: Inappropriate [OE specific]

Index: intltool-0.40.6/configure.in
===================================================================
--- intltool-0.40.6.orig/configure.in	2011-08-31 11:21:13.061160535 +0100
+++ intltool-0.40.6/configure.in	2011-08-31 11:22:44.991160885 +0100
@@ -9,24 +9,10 @@
 aclocaldir='${datadir}'/aclocal
 AC_SUBST(aclocaldir)
 
-AC_PATH_PROG(PERL, perl)
 if test -z "$PERL"; then
-   AC_MSG_ERROR([perl not found])
-fi
-AC_MSG_CHECKING([for perl >= 5.8.1])
-$PERL -e "use 5.8.1;" > /dev/null 2>&1
-if test $? -ne 0; then
-   AC_MSG_ERROR([perl 5.8.1 is required for intltool])
-else
-   PERL_VERSION="`$PERL -e \"printf '%vd', $^V\"`"
-   AC_MSG_RESULT([$PERL_VERSION])
-fi
-AC_MSG_CHECKING([for XML::Parser])
-if `$PERL -e "require XML::Parser" 2>/dev/null`; then
-   AC_MSG_RESULT([ok])
-else
-   AC_MSG_ERROR([XML::Parser perl module is required for intltool])
+   AC_MSG_ERROR([PERL variable not set])
 fi
+AC_SUBST(PERL)
 
 AC_OUTPUT([
 Makefile
